fdbclient/ManagementAPI.actor.cpp

```diff
@@ -1385,6 +1385,140 @@ ACTOR Future<Void> forceRecovery (Reference<ClusterConnectionFile> clusterFile)
 	}
 }
 
+json_spirit::Value_type normJSONType(json_spirit::Value_type type) {
+	if (type == json_spirit::int_type)
+		return json_spirit::real_type;
+	return type;
+}
+
+void schemaCoverage( std::string const& spath, bool covered ) {
+	static std::set<std::string> coveredSchemaPaths;
+	if (coveredSchemaPaths.insert(spath).second) {
+		TraceEvent ev(SevInfo, "CodeCoverage");
+		ev.detail("File", "documentation/StatusSchema.json/" + spath).detail("Line", 0);
+		if (!covered)
+			ev.detail("Covered", 0);
+	}
+}
+
+bool schemaMatch( StatusObject const schema, StatusObject const result, Severity sev, bool printErrors, bool checkCoverage, std::string path, std::string schema_path ) {
```

Comment:
It feels a little weird to me to have schemaMatch printing the errors for fdbcli. Maybe you could return error details for fdbcli to print out?